.circleci/config.yml

```diff
@@ -158,6 +158,7 @@ commands:
           command: |
             ./emcc --clear-cache
       - pip-install
+      - run: apt-get install ninja-build
```

Comment:
Hmm.. is ninja not part of our base imagine?   We should really update the image instead of doing this, but I guess that can be followup?